app/views/devise/unlocks/new.html.erb

```diff
@@ -1,16 +1,18 @@
-<h2>Resend unlock instructions</h2>
+<h2><%= link_to t(".title") %></h2>
```

Comment:
**nitpick (blocking):** Is their a reason why you are using `link_to` here or was it just a bad copy/paste.

Reply:
Yup, lots of translations, bad copy & pasting. Please see 25dbf61e43409bd3e907ae96673d9639764591d9.